vendor/README.md

```diff
@@ -319,7 +319,7 @@ This is a list of grammars that Linguist selects to provide syntax highlighting
 - **R:** [textmate/r.tmbundle](https://github.com/textmate/r.tmbundle)
 - **RAML:** [atom/language-yaml](https://github.com/atom/language-yaml)
 - **RDoc:** [joshaven/RDoc.tmbundle](https://github.com/joshaven/RDoc.tmbundle)
-- **REALbasic:** [angryant0007/VBDotNetSyntax](https://github.com/angryant0007/VBDotNetSyntax)
+- **REALbasic:** [jimmckay/XojoSyntaxTM](https://github.com/jimmckay/XojoSyntaxTM)
```

Comment:
Hm. The submodule wasn't changed. Did you use the appropriate script (see contribution guidelines) to change the grammar?

Reply:
I tried running the following, but the module doesn't exist.
```
$ script/add-grammar --replace REALbasic https://github.com/jimmckay/XojoSyntaxTM
Checking docker is installed and running
$ docker ps
Submodule 'vendor/grammars/REALbasic' does not exist. Aborting.
```
Not sure what the correct way to handle this is.

Reply:
It might be expected the name of the submodule for that argument:
```
$ script/add-grammar --replace VBDotNetSyntax https://github.com/jimmckay/XojoSyntaxTM 
```
To rename it, you might have to delete the current grammar and add one back with the `add-grammar` script. @Alhadis Do you confirm?

Reply:
VBDotNetSyntax isn't being replaced. It will still be used by Visual Basic files. REALbasic already exists and uses the VBDotNetSyntax. I'm only trying to change REALbasic to use the XojoSyntaxTM module as Xojo and REALbasic are the same format, only with differing file name extensions. Would it make more sense to remove REALbasic all together and just add the file extensions to Xojo since they are actually the same thing?

Reply:
> To rename it, you might have to delete the current grammar and add one back with the `add-grammar` script. @Alhadis Do you confirm?

Yes. The argument to `--replace` must match the name of a submodule folder within `./vendor/grammars/`.

Reply:
> I'm only trying to change REALbasic to use the XojoSyntaxTM module as Xojo and REALbasic are the same format, only with differing file name extensions.

Then you need to add a new grammar, still using `add-grammar`. See the contribution guidelines on adding a new language for help.